src/components/accessories/admission/PatientAdmission.tsx

```diff
@@ -47,6 +47,16 @@ const PatientAdmission: FC = () => {
     (state) => state.admissions.createAdmission.status
   );
 
+  const createError = useSelector<IState>(
```

Comment:
Instead of casting the returning value, you can do something like the following:

```suggestion
  const createError = useSelector<IState, string>(
```